list-pr-files/src/index.js

```diff
@@ -0,0 +1,27 @@
+const core = require("@actions/core");
+const github = require("@actions/github");
+
+async function main() {
+  try {
+    const token = core.getInput("ghToken");
+    const githubClient = new github.GitHub(token);
+
+    const contextPayload = github.context.payload;
+
+    const endpointOptions = githubClient.pulls.listFiles.endpoint.merge({
+      owner: contextPayload.repository.owner.login,
+      repo: contextPayload.repository.name,
+      pull_number: contextPayload.pull_request.number,
+    });
+
+    const files = await githubClient.paginate(endpointOptions, (response) =>
+      response.data.map((file) => file.filename)
+    );
+
+    core.setOutput("files", JSON.stringify(files));
```

Comment:
It might be much easier to work with if it were space separated list -- or if that is an option you can set.

JSON stringified can be read using jq?

Reply:
> JSON stringified can be read using jq?

I think so.

> It might be much easier to work with if it were space separated list

Files sometimes have spaces, how would one work around that?

Reply:
In bash, with quotes. But if jq can read the output, it's fine!